### PR TITLE
make me a gamer master ^w^

### PR DESCRIPTION
**Pull request by Tumesuo**

### CODE REVIEW

**Comment by Tumesuo** [comment]
I got an upvote from one of the headmins do it now

**Comment by MrDoomBringer** [comment]
image
i can't believe tumesuo is gonna become a game master!!

**Comment by Twaticus** [comment]
add me to this or im downvoting

**Comment by Tumesuo** [comment]
ok wait

**Comment by PolishUser** [comment]
Same please

File: config/admins.txt [hunk 1]
```diff
@@ -141,4 +141,7 @@ Skoglol = Game Master
 4dplanner = Game Master
 Time-Green = Game Master
 StyleMistake = Game Master
+tumesuo = Game Master
+twaticus = Game Master
+PolishUser = Game Master
```
Comment (Tlaltecuhtli) [inline config/admins.txt:146]:
```suggestion
PolishUser = Game Master
Tlaltecuhtli = Game Master
```

**Comment by Tumesuo** [comment]
Who do i request review from 😳

**Comment by Archanial** [comment]
fix travis smh

**Comment by MrDoomBringer** [comment]
please make me a game master..? pls??
also mods can we get a good first PR label

**Comment by Time-Green** [comment]
You're mocking the art of sneaking yourself into the admins.txt

**Comment by AffectedArc07** [comment]
oy wheres my fucking admin

**Comment by Akrilla** [comment]
bad cat *spank

**Comment by Tumesuo** [comment]
N-Nyoh~
*moan

**Comment by ArcaneMusic** [comment]
THIS IS HERESY OF THE HIGHEST ORDER.
EXTERMINATUS WILL CLEANSE THIS WORLD OF 7 BILLION.

**Comment by Tumesuo** [comment]
damn

**Comment by Tharcoonvagh** [comment]
cring

**Pull request closed without merging**